ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/support/commands.js

```diff
@@ -541,4 +541,10 @@ Cypress.Commands.add('assignUser', () => {
             cy.get('[class="govuk-button"]').click()
         }
     })
-})
\ No newline at end of file
+})
+
+//Navigate To Filter Projects section
+Cypress.Commands.add('navigateToFilterProjects',() => {  
+    cy.get('[data-cy="select-projectlist-filter-expand"]').click();
+    cy.get('.govuk-details__text').should('be.visible');
```

Comment:
Please can we use data attributes, or ids as selectors rather than classes, that way we can be very specific about what we are selecting. 
This class in particular could be quite common, and could change or be added to the page for another reason which could break this test.